build.assets/Dockerfile

```diff
@@ -70,6 +67,11 @@ RUN apt-get update -y --fix-missing && \
     apt-get -y clean && \
     rm -rf /var/lib/apt/lists/*
 
+# Install gcloud SDK and Firestore emulator
+RUN curl -sSL https://sdk.cloud.google.com | bash
+ENV PATH="$PATH:/root/google-cloud-sdk/bin"
+RUN gcloud components install cloud-firestore-emulator beta
```

Comment:
what's `beta`? do we need it?

Reply:
According to Google docs the firestore emulator is still in beta. gcloud requires to install `beta` package if you want to use it: https://cloud.google.com/sdk/gcloud/reference/beta/emulators/firestore/start

Reply:
I added a comment why we need beta package.